core/src/main/java/com/linecorp/armeria/client/HttpResponseDecoder.java

```diff
@@ -215,6 +215,7 @@ public boolean tryWrite(HttpObject o) {
                 }
             } else if (o instanceof HttpData) {
                 logBuilder.increaseResponseLength(((HttpData) o).length());
+                logBuilder.writeResponseContentPreview((HttpData)o);
```

Comment:
`increaseResponseLength` and `writeResponseContentPreview` could be merged into a single method?